steve/util.py

```diff
@@ -216,18 +391,21 @@ def get_project_config_file_name():
     return SteveConfig.file_name
 
 
-def create_project_config_file(dir_path):
+def create_project_config_file(dir_path, cfg=None):
     """Creates a new config file in directory
 
     :param dir_path: directory in which to create the config file
+    :param cfg:      if set alternative from default SteveConfig
 
     :returns: config file basename
     """
-    SteveConfig().create(dir_path)
+    if cfg is None:
+        cfg = SteveConfig
+    cfg().create(dir_path)
 
 
 def get_from_config(cfg, key, section='project',
-                    error='"{key}" must be defined in steve.ini file.'):
+                    error='"{key}" must be defined in {name} file.'):
```

Comment:
This more or less emulates the behaviour of "ConfigParser.get()" and of course it is nice to have the error Raised to actual tell you what section/key it is using.  I don't think the routine add much over directly using cfg.get(), you have to to hand in cfg anyway.